deployments/scripts/deploy-rbac-staging.sh

```
@@ -0,0 +1,239 @@
+    echo "To view pod status:"
+    echo "  kubectl get pods -n $NAMESPACE -l app=rbac-token-service"
+    echo
+    echo "Bootstrap admin token is stored in secret 'rbac-secrets'"
+    echo "To retrieve it:"
+    echo "  kubectl get secret rbac-secrets -n $NAMESPACE -o jsonpath='{.data.admin-bootstrap-token}' | base64 -d"
+}
+
+# Cleanup function for error handling
+cleanup() {
+    if [[ ${PORT_FORWARD_PID:-} ]]; then
+        kill $PORT_FORWARD_PID 2>/dev/null || true
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)